src/resources/sparkConf.properties

```diff
@@ -40,6 +40,8 @@ spark.source.writeTimeStampFilter                       false
 spark.source.minWriteTimeStampFilter                    0
 spark.source.maxWriteTimeStampFilter                    9223372036854775807
 
```

Comment:
@faizalrub-datastax Can we the default value "0" in this template props file? Also, can this prop be group with with other "spark.destination" around line 15

Reply:
Sure, applied changes!